src/client/Microsoft.Identity.Client/Platforms/Features/RuntimeBroker/RuntimeBroker.cs

```diff
@@ -135,7 +135,7 @@ private void LogEventRaised(NativeInterop.Core sender, LogEventArgs args)
             {
                 throw new MsalClientException(
                     "window_handle_required",
-                    "Desktop applications wanting to use the broker need to provide their window handle. See https://aka.ms/msal-net-wam#parent-window-handles");
+                    "Desktop applications now need to provide parent window handle to the broker. See https://aka.ms/msal-net-wam#parent-window-handles");
```

Comment:
```suggestion
                    "The new WAM implementation now needs application developers to provide parent window handle to the broker. See https://aka.ms/msal-net-wam#parent-window-handles");
```

something like this? Or can we get Den to write this up?